test/functional/DDR_Test/tck_ddrext.xml

```diff
@@ -113,13 +149,98 @@
-			</exec>
+		<echo>Moving core file from MVS [${dump.name}${zos.core.suffix}] to HFS [${system.dump}]</echo>
+		<exec executable="/bin/mv" failonerror="true">
+			<arg line="&quot;//${dump.name}${zos.core.suffix}&quot; ${system.dump}"/>
+		</exec>
 	</target>
 
 	<target name="clean" depends="TCK.run.tests.ddrext" description="clean">
 		<delete file="${system.dump}" />
 	</target>
-</project>
\ No newline at end of file
+
+	<!--	<target name="TCK_ddrext_deadlock" depends="clean">
+			<echo>This target will run deadlock detection DDR Extension tests </echo>
+			<antcall target="TCK.ddrext.deadlock">
+				<propertyset refid="propertyset.tck.ddrext" />
+		</antcall>
+	</target>-->
+
+	<target name="ddrext.setup" description="Create the required directories">
+		<!-- do not invoke this target with antcall as the properties defined here need to be visible to all subsequent targets -->
+		<echo>Setting up TCK</echo>
+		<mkdir dir="${tck.output.dir}"/>
+		<mkdir dir="${tck.output.dir}_${tck.type}"/>
+		<echo>Recording Java version</echo>
+		<java fork="true" jvm="${sdk.executable}${sdk.executable.ext}" classname="com.ibm.dtfj.tck.harness.TCKLauncher" 
+			failonerror="true" outputproperty="jvm.version" newenvironment="true">
+			<jvmarg value="-version" />
+		</java>
+		<condition property="java.is64bit">
+			<matches pattern=" jvm\S+64| p\S+64|\-64 " string="${jvm.version}" multiline="true"/>
+		</condition>
+
+		<!--real-time specific setup-->
+		<java fork="true" jvm="${sdk.executable}${sdk.executable.ext}" classname="com.ibm.dtfj.tck.harness.TCKLauncher" 
+			failonerror="false" outputproperty="jvm.version.realtime" newenvironment="true">
+			<jvmarg value="-Xrealtime" />
+			<jvmarg value="-version" />
+		</java>
+		<condition property="env.isrealtime">
+			<matches pattern=" real-time" string="${jvm.version.realtime}" multiline="true"/>
+		</condition>
+
+		<!-- z/OS specific setup -->
+		<condition property="tck.zos.core.suffix" value=".X001" else="">
+			<and>
+				<os family="z/os" />
+				<isset property="java.is64bit" />
+			</and>
+		</condition>
+	</target>
+
+	<target name="ddrext.setup.dump.dynamic" description="Create dump options for dynamic ddr ext test">
+		<!-- create dump options, different for z/OS -->
+		<condition property="system.dump" value="${user.name}.${tck.core.file.mvs}" else="${tck.core.file.ddrext}">
+			<os family="z/os" />
+		</condition>
+
+		<condition property="isWindows">
+			<os family="windows" />
+		</condition>
+		<condition property="isZOS">
+			<os family="z/os" />
+		</condition>
+
+		<var name="temp.tck.config.dump.opt" value = ""/>
+
+		<if>
+			<isset property="isZOS"/>
+			<then>
+				<var name="temp.tck.config.dump.opt" value="-Xdump:system:defaults:request=exclusive+compact+prepwalk,label=${user.name}.${tck.core.file.mvs}"/>
+			</then>
+			<elseif>
+				<isset property="isWindows"/>
+				<then>
+					<tempfile property="path.file"/>
```

Comment:
A file is not needed for this; `<pathconvert/>` is probably what you want here.
See https://ant.apache.org/manual/Tasks/pathconvert.html

Reply:
I changed into `<pathconvert>` now, could you please take a look?

Reply:
I changed into `<pathconvert>` now, could you please take a look? @keithc-ca

Reply:
It's actually even simpler if we use `<property ... location="..."/>` which does the right thing on Windows and other platforms:
```
<if>
	<isset property="isZOS" />
	<then>
		<property name="system.dump.location" value="${user.name}.${tck.core.file.mvs}" />
	</then>
	<else>
		<property name="system.dump.location" location="${system.dump}" />
	</else>
</if>
<property name="tck.config.dump.opt.1" value="-Xdump:system:defaults:label=${system.dump.location},request=exclusive+compact+prepwalk" />
```

Reply:
Actually, because `ddrext.setup.dump.dynamic` sets `system.dump` properly on z/OS, this doesn't need to make a special case and can be further simplified:
```
<property name="system.dump.location" location="${system.dump}" />
<property name="tck.config.dump.opt.1" value="-Xdump:system:defaults:label=${system.dump.location},request=exclusive+compact+prepwalk" />
```